x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java

```diff
@@ -138,10 +138,11 @@ public synchronized AutoFollowStats getStats() {
     }
 
     synchronized void updateStats(List<AutoFollowResult> results) {
+        long newStatsReceivedTimeStamp = relativeMillisTimeProvider.getAsLong();
```

Comment:
We want the absolute time provider here!

Reply:
whoops, you're absolutely right. I guess I made it this change too quickly...